src/rewrite-missing.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<style>body{background-color:#f1f5f9;color:#1d273b;letter-spacing:0;min-height:100%;position:relative;font-family:Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.4;margin:0}.error{justify-content:center;display:flex;max-width:30rem;margin-left:auto;margin-right:auto;width:100%;padding:5rem;align-items:center;flex-direction:column;height:100%;text-align:center}.error-header{color:#616876;font-size:4rem;font-weight:300;line-height:1;margin:0 0 1rem}.error-message{margin:0 0 .5rem;font-size:1.25rem;line-height:1.75rem}.error-help{margin:0 0 .5rem;color:#616876}a{color:#206bc4;text-decoration:none}</style>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<title>Fatal Error: Missing Required HTTPd Module</title>
+	</head>
+	<body>
+		<div class="error">
+			<h1 class="error-header">Fatal Error</h1>
+			<p class="error-message">Missing Required HTTPd Module</p>
+			<p class="error-help">FOSSBilling requires mod_rewrite to operate correctly when using Apache, and other .htaccess compatible web servers.</p>
```

Comment:
Or point the user to use: https://github.com/FOSSBilling/FOSSBilling/blob/116f55851f62cd28faa5627f13e7413584ce7bd2/src/config-sample.php#L94

Reply:
At this stage, I'm not even sure if FOSSBilling works with that setting to false. 

It seems that we've accepted rewrite as a base requirement - but I'm not sure - maybe non-friendly URLs should still be supported?

Reply:
sef_urls to false is the default if mod rewrite is enabled. 

Setting it to true changes to to non search engine friendly urls :)

Reply:
Jaap, that's quite the opposite.

Reply:
`sef_urls` will come enabled by default no matter anything. It can be disabled afterwards, but I don't really see why anyone would want to do that.

Reply:
If they don't support modrewrite personally no reason anyways.